openid.html

```diff
@@ -87,11 +94,13 @@
         var v1 = $("#node-config-input-discovery").val();
         var v2 = $("#node-config-input-clientId").val();
         var v3 = $("#node-config-input-clientSecret").val();
-        $("#node-config-start-auth").toggleClass("disabled",(v1.length === 0 || v2.length === 0 || v3.length === 0));
+        var v4 = $('#node-config-input-scopes').val();
+        $("#node-config-start-auth").toggleClass("disabled",(v1.length === 0 || v2.length === 0 || v3.length === 0 || v4.length === 0));
```

Comment:
As you have put a placeholder in the HTML form, scope should not be mandatory.

```suggestion
        $("#node-config-start-auth").toggleClass("disabled",(v1.length === 0 || v2.length === 0 || v3.length === 0));
```